src/data/faq.json

```diff
@@ -293,12 +293,15 @@
                         ]
                     },
                     {
-                        "title": "Can I use a complete vaccination status in the app as a travel permission?",
+                        "title": "Can I use my vaccination certificate in the app as a travel permission?",

```

Comment:
Suggest you use "a vaccination certificate" not "my vaccination certificate" since the app can store vaccination certificates for multiple people, not only for myself.